### ISSUE TITLE
Facebook banner ad shows additional margin at top

### ISSUE THREAD

**Issue by girishjjain-rh** [expo/expo#8237]
### Environment

  Expo CLI 3.20.3 environment info:
    System:
      OS: macOS 10.15.4
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 12.10.0 - /usr/local/bin/node
      Yarn: 1.17.3 - /usr/local/bin/yarn
      npm: 6.14.4 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Android Studio: 3.5 AI-191.8026.42.35.5900203
      Xcode: 11.4/11E146 - /usr/bin/xcodebuild
    npmGlobalPackages:
      expo-cli: 3.20.3

Target: iOS, Android

### Steps to Reproduce
Use following code to reproduce issue:
```
import React from 'react';
import { StyleSheet, View, SafeAreaView } from 'react-native';
import * as FacebookAds from 'expo-ads-facebook';

export default class App extends React.Component {

  render() {
    FacebookAds.AdSettings.addTestDevice(FacebookAds.AdSettings.currentDeviceHash);
    return (
      <SafeAreaView style={styles.container}>
        <View style={{backgroundColor:'yellow'}}>
          <FacebookAds.BannerAd
            placementId="IMG_16_9_APP_INSTALL#2890122774429035_2890124944428818"
            type="standard"
            onPress={() => console.log('click')}
            onError={error => console.log('error', error)}
          />
        </View>
      </SafeAreaView>
    );  
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    flexDirection: 'column',
  },
});
```

### Expected Behavior
Banner ad to display without additional top margin

### Actual Behavior
Banner Ad displays with additional top margin (shown in yellow background in screenshots). Refer below screenshots with element inspector that shows two components being rendered for banner ad:
**Screenshot 1:** Banner ad component is selected with element selector and the additional yellow background above it. Notice position of component:
![Simulator Screen Shot - iPhone 11 Pro Max - 2020-05-10 at 07 24 00](https://user-images.githubusercontent.com/40406953/81499672-c4a13080-9292-11ea-8e3f-4408e451cd6b.png)

**Screenshot 2:** Selected the area with yellow background using element inspector, Notice position of selected component vs banner ad component from Screenshot 1:
![Simulator Screen Shot - iPhone 11 Pro Max - 2020-05-10 at 07 23 47](https://user-images.githubusercontent.com/40406953/81499673-c539c700-9292-11ea-9d47-454bc67081f9.png)

### Reproducible Demo
https://snack.expo.io/@girishjjain/5e06a

**Comment by cruzach** [comment]
Hey @girishjjain-rh - what SDK are you using? and does this layout issue happen on both ios and android?

**Comment by girishjjain-rh** [comment]
Hi @cruzach I am using SDK 37. It happens on iOS for sure, didn't check Android yet. Will report back here once I do.

**Comment by girishjjain-rh** [comment]
Not able to load any Ad on Android. Looks like for Android platform the Facebook Audience Network SDK used is very old. Facebook Audience Network SDK current version is v5.9 (which is updated for iOS) but for Android Expo is using v5.1. Refer - https://github.com/expo/expo/blob/master/packages/expo-ads-facebook/android/build.gradle#L61

**Comment by girishjjain-rh** [comment]
@cruzach Do you need more info for this issue?

**Comment by cruzach** [comment]
It would be useful to know if the same behavior happens in a bare workflow app

**Comment by artemagvanian** [comment]
I have faced the same problem in the bare workflow on the iOS platform and it looks like I found the code responsible for this. 

In "packages/expo-ads-facebook/ios/EXAdsFacebook/EXBannerView.m" on line 51:
`adView.frame = CGRectMake(0, 20, adView.bounds.size.width, adView.bounds.size.height);`

For some reason, the rectangle of the ad is being created with y-coordinate = 20. I changed it to 0 and it works for me.

**Comment by girishjjain-rh** [comment]
That helps @artemagvanian Thanks!

@cruzach - For Android platform, old version (v5.1) of Facebook Audience Network SDK is being used which is more than a year old. I believe latest is v5.9. I hope that gets fixed as part of  the upcoming Expo SDK release. Do you think if it needs to be logged as a separate issue or has it already made it to team's backlog?

**Comment by abudhia-oyo** [comment]
Hi, just checked with Expo SDK 39, this is still present. Expo Ads Facebook cannot be used for new Facebook Apps due to unsupported FB SDK Version Issue.

Completely blocked on this.

**Comment by cruzach** [comment]
Yes, both versions will be upgraded in this PR - https://github.com/expo/expo/pull/10430

**Comment by cruzach** [comment]
@girishjjain this fix was not included in the SDK 39 patch release

**Comment by girishjjain** [comment]
Thanks @cruzach for confirming that. The timing of this PR and patch release made me think this might have been included in the patch release. Do you know if we would have to wait until next Expo SDK release for this fix to be available for managed workflow?

**Comment by cruzach** [comment]
Yes, you'd have to wait for the next SDK because I think we don't want people "accidentally" receiving this fix if they've already upgraded to SDK 39. 

Most likely anyone that's run into this has made up for it in the styling of their component (with a negative margin or something like that), so if we update this in the middle of an SDK the UI of their app will have changed and it won't be obvious why. It's more likely they'll be able to trace the change back to this if the fix is released as part of a major SDK release, so I think we'd prefer to do that. Does that make sense?

**Comment by girishjjain** [comment]
That makes sense, thanks @cruzach for responding!
